benchmarks/dynamo/ci_expected_accuracy/cpu_inductor_timm_amp_freezing_inference.csv

```diff
@@ -110,7 +110,7 @@ jx_nest_base,pass,0
 
 
 
-lcnet_050,pass,0
+lcnet_050,fail_accuracy,0
```

Comment:
Why?

Reply:
lcnet_050 and doctr_reco_predictor will fail accuracy when conv_bn folding is done. It's a known issue, the [PR](https://github.com/pytorch/pytorch/pull/129775/files) make them pass by adding them in `inline_inbuilt_nn_modules_models `list, this PR supports Conv BN folding for inline inbuilt modules, so the issue is back.